test/deploy/deploy_unittest.sh

```diff
@@ -1,13 +1,17 @@
 #!/bin/bash
-git clone git@github.com:dmwm/deployment.git
-(cd deployment; curl https://github.com/dmwm/deployment/commit/3309be9ce3b628efe39e1db30b6a1f865e1fa3e4.patch | git am)
+WMA_TAG=1.0.7.pre3
+git clone https://github.com/dmwm/deployment.git
```

Comment:
dmwm should be a configurable variable too. I want to check out my fork so I can make changes.